src/react/databrowser/buckets/details/Overview.tsx

```diff
@@ -97,15 +136,27 @@ function Overview({ bucket, ingestionStates }: Props) {
   const workflowsQuery = useWorkflows();
```

Comment:
actually we can pass `filters` which is a list of bucket name to filter the workflows directly.
```suggestion
  const workflowsQuery = useWorkflows([bucket.Name]);
```